edk2toolext/tests/test_secureboot_audit.py

```diff
@@ -0,0 +1,133 @@
+# @file test_repo_resolver.py
+# This contains unit tests for repo resolver
+#
+##
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import os
+import unittest
+import json
+import tempfile
+
+from edk2toolext.windows.secureboot.secureboot_audit import (
+    generate_dbx_report,
+    filter_revocation_list_by_arch,
+    convert_uefi_org_revocation_file_to_dict,
+    write_xlsx_file,
+    write_json_file,
+)
+
+
+# Setup the test directory path
+TEST_DATA_PARENT_DIRECTORY = os.path.join(
```

Comment:
So orignally the weren't meant for testing but they have become that. I'll go ahead and move them

Reply:
Most people are going to access your tool by pip installing edk2-pytool-examples and using the command via the CLI. They won't be able to access the these files as is. If they are going to actually review the readme on how to use the tool, it will be via the documentation we automatically generate (See [example](https://www.tianocore.org/edk2-pytool-extensions/tools/using_image_validation_tool/)). Due to this, if you want to show an easily accessible example, I would take a screenshot, lower the quality of the image, and put it in your readme.